app.yaml

```diff
@@ -0,0 +1,26 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+- url: /(.+)
```

Comment:
I am a bit confused about the conventions here. Isn't our entire website (so far) completely static? If so, let's just put the whole thing in a single static/ directory.

Reply:
I did moved everything under `/app` to be uploaded to GCP. 
You can take a look at the new directory structure [here](https://github.com/oppia/foundation-website/tree/34afc76eadabc1ef8c76f3788c09a2150a571f65)